kubernetes/users/discourse-staging-rbac.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Should this file be renamed to `discourse-stage-rbac.yaml`?

Reply:
yes, good catch!